Source/BusinessMomentsAdaptorImage/rawdata/for_rawdataProcessor/when_handling_data_from_platform.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+import { expect } from 'chai';
+import fs from 'fs';
+import path from 'path';
+import { EntityTransformer } from '../processor';
+
+describe('handling data from Platform', () => {
+
+    let payloadFromPlatform: any;
+
+    beforeEach(() => {
+        // just for learning purposes
+        const testDataDir = `${path.resolve(__dirname)}/data`;
+        const data = fs.readFileSync(`${testDataDir}/data-from-platform.json`);
+        payloadFromPlatform = JSON.parse(data.toString());
+    });
+
+    it('should be possible to read json file', () => {
+        expect(payloadFromPlatform).to.not.be.undefined;
+        expect(payloadFromPlatform.name).to.equal('Webhook-101');
+    });
+
+    it('will transform payload to EntityTransformers', () => {
+        const platformDataHandler = new PlatformDataHandler();
+        const transformers = platformDataHandler.ExtractTransformers(payloadFromPlatform);
+
+        expect(transformers.length).is.greaterThan(0);
+        expect(transformers[0].Name).to.equal('I am name');
+        expect(transformers[0].Filter).to.not.be.undefined;
+        expect(transformers[0].Filter({ system: 'M3' })).to.equal(true);
+        expect(transformers[0].Filter({ system: 'IFS' })).to.equal(false);
+    });
+
+});
+
+// TODO: refactor this into a separate file
+class PlatformDataHandler {
+
+    ExtractTransformers(input: any): EntityTransformer[] {
+        const result = new Array<EntityTransformer>();
+        for (const entityDefinition of input.extra.entities) {
+            result.push({
+                Name: entityDefinition.name,
+                Filter: new Function(`return (${entityDefinition.filter});`)() as (data: any) => boolean,
```

Comment:
I believe ".filter" and ".transform" need to become ".filterCode" and ".transformCode".